doc/modules/ROOT/pages/tutorials/visualize-with-pyvis.adoc

```diff
@@ -0,0 +1,201 @@
+// DO NOT EDIT - AsciiDoc file generated automatically
+
+= GDS Projection Visualization with PyVis
```

Comment:
How about mentioning PyVis only later in a section? Makes it easier to extend with other visualization libs

Reply:
Sure :)